pages/NavigationPages.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from './css/NavigationPage.css';
```

Comment:
I think this needs to be `../css/...` as you've got the same style in two places